shared/common-adapters/copyable-text.native.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import _ from 'lodash'
+import React, {Component} from 'react'
+import {Clipboard, TouchableHighlight} from 'react-native'
+import Box from './box'
+import Text from './text'
+import HOCTimers from './hoc-timers'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props as PropsCommon} from './copyable-text'
+import type {TimerProps} from './hoc-timers'
+
+export type Props = PropsCommon & {
+  extras: Object,
```

Comment:
what is this extras?

Why not just make it style specific? I don't think we need a way to insert arbitrary propts to text, right?